.azure-pipelines/client.test.storage.live.yml

```diff
@@ -0,0 +1,155 @@
+# External variables:
+# Package
+#   The package to build and test. Defined in pipeline web ui because multiple pipelines use this YAML with different packages.
+#   Example: eventhubs-client
```

Comment:
While I think we can keep the `$(Package)` variable to help with merging and keeping it consistent with the primiary live test yml file I think we should consider setting it to Storage directly in the yml since I don't think this yml will work with any other live testing package.

Reply:
We do have three independent packages for storage. `$(Package)` can be
- `storage-blob` or
- `storage-file` or
- `storage-queue`

All three packages will use the same configurations/setup as in client.test.storage.live.yml.
The other packages like `eventhubs-client`, `eventhubs-processor` and `servicebus` will use client.test.live.yml.

Should we consider having only one build pipeline for all the storage-packages ?

Reply:
Good Point. We need to decide how we would configure the nightly scheduled live test job as you cannot create scheduled builds with different parameters so you would need to have a unique build pipeline for each one which is also not a great approach either. I would suggest we figure out a way to run all the storage live tests together if they are going to share this yml.

Reply:
Although I just checked and it looks like @mikeharder already created one for each storage package so I'm Ok sticking with that pattern for now.